controller/src/main/java/io/pravega/controller/server/rpc/auth/PravegaInterceptor.java

```diff
@@ -69,6 +69,8 @@
                 context = context.withValue(AUTH_CONTEXT_PARAMS, paramMap);
                 context = context.withValue(INTERCEPTOR_OBJECT, this);
             }
+        } else {
+            call.close(Status.fromCode(Status.Code.UNAUTHENTICATED), headers);
```

Comment:
As I understand it, this line is setting the status code to `UNAUTHENTICATED`, but the description of the PR says:

```
Throw exception when authentication method is not selected.
```

It seems to be talking about the caller's code path and not the change in the PR. Please clarify and consider providing a more precise explanation.

Reply:
Updated the PR comment.